source/com.microsoft.tfs.client.common.ui/src/com/microsoft/tfs/client/common/ui/teamexplorer/internal/TeamExplorerHelpers.java

```diff
@@ -56,6 +69,25 @@
 
     public static final String EGitRepoViewID = "org.eclipse.egit.ui.RepositoriesView"; //$NON-NLS-1$
 
+    private static final String GIT_TOKEN = "_git"; //$NON-NLS-1$
+    private static final String DEFAULT_COLLECTION_TOKEN = "DefaultCollection"; //$NON-NLS-1$
+
+    private final static String PROTOCOL_HANDLER_ARG = "-openuri"; //$NON-NLS-1$
+    private final static String PROTOCOL_HANDLER_SCHEME = "vsoi"; //$NON-NLS-1$
```

Comment:
This is temporary, for dev. purposes. I'll replace it with vsoe as soon as we add the corresponding featore to the server.